website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Apache APISIX 2.10.0 正式发布，带来第一个 LTS 版本！"
```

Comment:
Do we need this duplicate post? I see there has one `website/blog/2021-09-29-release-apache-apisix-2.10.md`

Reply:
yes We can remove it